app/main.py

```diff
@@ -1,37 +1,67 @@
 import json
+import logging
 import os
+from pathlib import Path
 
+from colorlog import ColoredFormatter
 from load_dotenv import load_dotenv
 
-from app.llm_api_client import LlmApiClient
+from app.llm_api_client import LlmApiClient, Model
 from app.parser import Parser
+from app.prompt_builder import PromptBuilder
 from app.reviewer import Reviewer
 from dtos import CriteriaDto
 
 
+def set_logger():
+    formatter = ColoredFormatter(
+        "%(log_color)s%(asctime)s - %(levelname)-8s%(reset)s %(blue)s%(message)s",
+        datefmt="%Y-%m-%d %H:%M:%S",
+        reset=True,
+        log_colors={
+            'DEBUG': 'cyan',
+            'INFO': 'green',
+            'WARNING': 'yellow',
+            'ERROR': 'red',
+            'CRITICAL': 'red,bg_white',
+        }
+    )
+
+    handler = logging.StreamHandler()
+    handler.setFormatter(formatter)
+
+    logger = logging.getLogger()
+    logger.addHandler(handler)
+    logger.setLevel(logging.INFO)
+
+
 def main():
+    set_logger()
+
     load_dotenv()
     api_key = os.getenv("OPENAI_API_KEY")
-    model_name = os.getenv("OPENAI_MODEL_NAME", "gpt-3.5-turbo")
 
-    llm_api_client = LlmApiClient(api_key=api_key, model_name=model_name)
+    llm_api_client = LlmApiClient(api_key=api_key, model_name=Model.GPT_4o)
     parser = Parser()
-    reviewer = Reviewer(llm_api_client=llm_api_client, parser=parser)
+    prompt_builder = PromptBuilder("../prompts")
+    reviewer = Reviewer(llm_api_client=llm_api_client,
+                        parser=parser,
+                        prompt_builder=prompt_builder)
 
-    paths_to_code = "/Users/mikhailkharlamov/Documents/Projects/SummerSchool/quality-linter/dataset/file_"
+    paths_to_code = Path("/Users/mikhailkharlamov/Documents/Projects/SummerSchool/quality-linter/dataset/file_")
```

Comment:
there is no such path, nothing exists at `file_`) I propose:

```suggestion
    dataset_path = Path("/Users/mikhailkharlamov/Documents/Projects/SummerSchool/quality-linter/dataset")
```

and then to use it:
```python
path = dataset_path / f"file_{i}"
```